api/app/registration/api.py

```diff
@@ -143,30 +143,29 @@ def post(self):
         expiry_date = datetime.strptime((args['expiry_date']), '%Y-%m-%dT%H:%M:%S.%fZ')
         payment_required = args['payment_required']
         travel_award = args['travel_award']
-        accepted_accommodation_award = args['accepted_accommodation_award']
-        accepted_travel_award = args['accepted_travel_award']
         accommodation_award = args['accommodation_award']
         user = db.session.query(AppUser).filter(AppUser.id == user_id).first()
-
         event_name = db.session.query(Event).filter(Event.id == event_id).first().name
 
+        existing_offer = db.session.query(Offer).filter(Offer.user_id == user_id, Offer.event_id == event_id).first()
+        if existing_offer:
+            return errors.DUPLICATE_OFFER
+
         offer_entity = Offer(
             user_id=user_id,
             event_id=event_id,
             offer_date=offer_date,
             expiry_date=expiry_date,
             payment_required=payment_required,
             travel_award=travel_award,
-            accommodation_award=accommodation_award,
-            accepted_accommodation_award=accepted_accommodation_award,
```

Comment:
Is this not working properly?

Reply:
It works - but we shouldn't be setting it when creating an offer...